src/Task/PhpStan.php

```diff
@@ -56,7 +60,17 @@ public function canRunInContext(ContextInterface $context)
     public function run(ContextInterface $context)
     {
         $config = $this->getConfiguration();
-        $files = $context->getFiles()->extensions($config['triggered_by']);
+        $files = $context
```

Comment:
Can you add tests for both `ignore_patterns` and `force_patterns` to make sure the tasks behaves as it should with those 2 additional parameters?